jdaviz/configs/imviz/tests/test_parser.py

```diff
@@ -378,7 +378,7 @@ def test_parse_jwst_niriss_grism(self, imviz_helper):
         expected_label = os.path.splitext(os.path.basename(self.jwst_asdf_url_2))[0] + '[DATA]'
         assert data.label == expected_label
         assert data.shape == (2048, 2048)
-        assert data.coords is None
+        assert data.coords is not None
```

Comment:
This data is https://stsci.box.com/shared/static/d5k9z5j05dgfv6ljgie483w21kmpevni.fits . I guess going from no WCS to having WCS is a good thing? Perhaps inspect whether the WCS makes any sense?

Reply:
I guess so! I'll try the data in a notebook and give it a look. Can't promise that the science results will make sense though.

Reply:
Does mouseover info make sense?